ModernFlyouts/Properties/Strings.resx

```diff
@@ -497,4 +497,10 @@
   <data name="Settings.VerticalAlignment" xml:space="preserve">
     <value>Vertical alignment</value>
   </data>
+  <data name="Settings.FlyoutAxis" xml:space="preserve">
+    <value>Change axis of flyout</value>
```

Comment:
"Flyout Axis:" as the header may be more concise

Reply:
Changed